CHANGELOG.md

```diff
@@ -1,6 +1,7 @@
 
 ### To Be Released...
 * Sons Of The Forest - Added support
+* Red Dead Redemption 2 - Added support
```

Comment:
Nitpicking a bit, make sure this matches the other files game name (missing the `- RedM (2018)` part)

Reply:
Valid